lib/qa/authorities/linked_data/find_term.rb

```diff
@@ -80,6 +80,7 @@ def perform_normalization
           return full_graph.dump(:jsonld, standard_prefixes: true) if jsonld?
 
           filter_graph
+          extract_uri
```

Comment:
Extracting the uri here forces the uri to be updated for every call to find based on the current graph represented in `filter_graph`.